frontend/src/console/components/PreviewComponent/__snapshots__/PreviewComponent.spec.tsx.snap

```diff
@@ -3,417 +3,5 @@
-                  }
-                }
-                type="button"
-              >
-                <span>
-                  Donate
-                </span>
-              </button>
-            </div>
-          </div>
-          <div
-            className="col-md-5"
-          />
-          <div
-            className="openedx-logo col-md-3"
-          >
-            <img
-              alt="Open edX logo"
-              src="openedx-logo-footer.png"
-            />
-          </div>
-        </div>
-        <div
-          className="row"
-        >
-          <div
-            className="logo col-md-1"
-          />
-        </div>
-        <div
-          className="row"
-        >
-          <div
-            className="copyright"
-            style={
-              Object {
-                "color": undefined,
-              }
-            }
-          >
-            © test. 
-            All rights reserved except where noted. edX, Open edX and their respective logos are registered trademarks of edX Inc.
-          </div>
-        </div>
-        <div
-          className="legal row"
-        >
-          <button
-            className="customizable-link noHover"
-            onMouseEnter={[Function]}
-            onMouseLeave={[Function]}
-            style={
-              Object {
-                "borderBottomColor": undefined,
-                "color": undefined,
-              }
-            }
-            type="button"
-          >
-            <span>
-              Privacy Policy
-            </span>
-          </button>
-          <button
-            className="customizable-link noHover"
-            onMouseEnter={[Function]}
-            onMouseLeave={[Function]}
-            style={
-              Object {
-                "borderBottomColor": undefined,
-                "color": undefined,
-              }
-            }
-            type="button"
-          >
-            <span>
-              Terms of Service
-            </span>
-          </button>
-          <button
-            className="customizable-link noHover"
-            onMouseEnter={[Function]}
-            onMouseLeave={[Function]}
-            style={
-              Object {
-                "borderBottomColor": undefined,
-                "color": undefined,
-              }
-            }
-            type="button"
-          >
-            <span>
-              Honor Code
-            </span>
-          </button>
-          <button
-            className="customizable-link noHover"
-            onMouseEnter={[Function]}
-            onMouseLeave={[Function]}
-            style={
-              Object {
-                "borderBottomColor": undefined,
-                "color": undefined,
-              }
-            }
-            type="button"
-          >
-            <span>
-              Take free online courses at edX.org
-            </span>
-          </button>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
```

Comment:
Could you please adjust tests for this component, so we're not removing the whole snapshot (include correct prop value)? Otherwise, this test makes no sense.

Reply:
True. I've added the consequent test case.